script/bootstrap

```diff
@@ -11,5 +11,5 @@ script/utils/launch-docker
 
 echo "==> Installing application dependencies..."
 
-nodenv install --skip-existing
-npm install
+npm_config_yes=true npx nodenv install --skip-existing
+npm i
```

Comment:
I'm still a little confused about this - are you able to document what's going on here and why this is needed in the commit message (same with the other changes in line with the ADR I linked in a separate comment)?

I didn't realise you could use `nodenv` via `npx`, but if we are going down that route (rather than installing via Homebrew), it looks like the [@nodenv/nodenv](https://www.npmjs.com/package/@nodenv/nodenv) package is the one, not [nodenv](https://www.npmjs.com/package/nodenv)?

Could we keep the more verbose `npm install` for readability?

Reply:
Homebrew is mac specific and we should be environment agnostic.

Good spot on @nodenv/... Shame the author didn't have his original deprecated as I didn't get a warning.

It should be able to disappear though, we have docker for disconnecting the environment and can just make a override for app in docker-compose that runs nodemon inside the container.

Reply:
Could we resolve the `npm i` -> `npm install` issue and document in the commit message what's going on/why with the change on the previous line?

I'm also a little unclear on what the changes have to do with a container name, which is mentioned in the commit message - can you elaborate? As far as I can tell, the Docker change is just a DRY up